GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest

```diff
@@ -36,9 +36,9 @@
         <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="VsPackage.TAfGT" Path="|VsPackage.TAfGT|" />
     </Assets>
     <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.8.27729.1,17.0)" DisplayName="Visual Studio core editor" />
-        <Prerequisite Id="Microsoft.VisualStudio.Component.VC.CoreIde" Version="[15.6.27406.0,17.0)" DisplayName="Visual Studio C++ core features" />
-        <Prerequisite Id="Microsoft.VisualStudio.Component.NuGet" Version="[15.9.28016.0,17.0)" DisplayName="NuGet package manager" />
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.8.27729.1,)" DisplayName="Visual Studio core editor" />
```

Comment:
I don't understand the meaning of these very well, so I just wanted to call out that these reference 15.8. I'm not sure if they need to likewise be updated to 17.0?

Reply:
I think we're okay if we always limit the version we're available to install for. I think they probably could be updated, but I think it's just as safe to leave them as is.